core/src/commonMain/kotlin/com/xebia/functional/xef/store/MemorizedMessage.kt

```diff
@@ -18,7 +18,7 @@ sealed class MemorizedMessage {
         ChatCompletionRequestMessage.CaseChatCompletionRequestAssistantMessage(
           ChatCompletionRequestAssistantMessage(
             role = ChatCompletionRequestAssistantMessage.Role.assistant,
-            content = message.content
+            content = message.content ?: message.toolCalls?.firstOrNull()?.toString()
```

Comment:
When we don't have content, we add the JSON of the `toolCalls` directly as `String` when the messages come from memory.

We have to find a better strategy for this.